go.mod

```diff
@@ -4,99 +4,102 @@ go 1.17
 
 require (
 	contrib.go.opencensus.io/exporter/prometheus v0.4.0
-	github.com/application-research/filclient v0.0.0-20220622165741-3ca6a3f3bc7a
-	github.com/application-research/go-bs-autobatch v0.0.0-20211215020302-c4c0b68ef402
-	github.com/cenkalti/backoff/v4 v4.1.2
+	github.com/application-research/go-bs-autobatch v0.0.0-20220803173502-846f018c8bf9
+	github.com/cenkalti/backoff/v4 v4.1.3
 	github.com/cheggaaa/pb/v3 v3.0.8
 	github.com/docker/go-units v0.4.0
 	github.com/dustin/go-humanize v1.0.0 // indirect
 	github.com/facebookgo/atomicfile v0.0.0-20151019160806-2de1f203e7d5
-	github.com/filecoin-project/boost v0.0.0-20220310131759-cd38741e464d
-	github.com/filecoin-project/go-address v0.0.6
+	github.com/filecoin-project/boost v1.2.1-0.20220803131342-cfcd4837136d
```

Comment:
v1.2.1 is the master branch of boost. We should use the latest boost release tag instead.